src/api/guzzle.rs

```diff
@@ -8,11 +9,22 @@ struct GuzzleResponse {
 
 const GUZZLE: &str = "https://api.mydadleft.me";
 
+pub static REQWEST_CLIENT: Lazy<reqwest::Client> = Lazy::new(|| {
+	reqwest::Client::builder()
+		.user_agent("teawieBot/0.1.0")
+		.build()
+		.unwrap_or_default()
+});
+
```

Comment:
oops, almost missed this :p

i think this be better placed in `src/api/mod.rs` since it's a bit odd to have it locked to the guzzle namespace when it can be used across all files